tree/tree/src/InternalTreeUtils.cxx

```diff
@@ -169,6 +173,9 @@ std::vector<std::string> GetFileNamesFromTree(const TTree &tree)
 /// \ingroup tree
 /// \brief Get and store the names, aliases and file names of the direct friends of the tree.
 /// \param[in] tree The tree from which friends information will be gathered.
+/// \param[in] retrieveEntries Whether to also retrieve the number of entries in
+///            each tree of each friend. In case of a TChain, this triggers
```

Comment:
"each tree of each friend" -> "each friend tree"?

Reply:
Rephrased to
```
/// \param[in] retrieveEntries Whether to also retrieve the number of entries in
///            each tree of each friend: one if the friend is a TTree, more if 
///            the friend is a TChain. In the latter case, this function
///            triggers the opening of all files in the chain.
```